### PR TITLE
Add Jackson parser to properly parse json file

### PR DESCRIPTION
**Pull request by cmoine**

### CODE REVIEW

**Comment by cmoine** [comment]
It addresses issue https://github.com/moshbit/Kotlift/issues/11

File: src/com/moshbit/kotlift/Main.kt [hunk 1]
```diff
@@ -53,6 +56,9 @@ fun main(args: Array<String>) {
     val destPath = Paths.get(file.path.replace(sourcePath, destinationPath).replace(".kt", ".swift"))
     destinationFiles.add(destPath.toFile())
 
+    val parentDir=destPath.parent.toFile()
```
Comment (vonox7) [inline src/com/moshbit/kotlift/Main.kt]:
```suggestion
    val parentDir = destPath.parent.toFile()
```

File: src/com/moshbit/kotlift/Main.kt [hunk 2]
```diff
@@ -53,6 +56,9 @@ fun main(args: Array<String>) {
...
+    if(!parentDir.exists())
+      parentDir.mkdirs();
```
Comment (vonox7) [inline src/com/moshbit/kotlift/Main.kt]:
```suggestion
    if(!parentDir.exists()) {
      parentDir.mkdirs()
    } 
```

**Review by vonox7 — COMMENTED** [review]
Nice :)

**Pull request merged**